confdb/aclocal_modules.m4

```diff
@@ -139,6 +139,7 @@ AC_DEFUN([PAC_CONFIG_HWLOC],[
         if test -n "$hwloc_darwin_ldflags" ; then
             echo "hwloc_darwin_ldflags = $hwloc_darwin_ldflags"
             PAC_APPEND_FLAG([$hwloc_darwin_ldflags], [LDFLAGS])
+            PAC_APPEND_FLAG([$hwloc_darwin_ldflags], [WRAPPER_LDFLAGS])
```

Comment:
We only need do this for static build, right? Can we add this condition?

Reply:
We could, but we also have the `-static-mpi` option in the wrapper scripts to support static linking `libmpi.a` even when shared libs are present. Maybe I should add them to `SAVE_LDFLAGS` since those are always used in the static case?